maker/src/routes.rs

```diff
@@ -140,6 +141,46 @@ pub async fn post_sell_order(
     Ok(())
 }
 
+/// The maker POSTs this to set the offer params
+#[derive(Debug, Clone, Deserialize)]
+pub struct CfdNewOfferParamsRequest {
+    pub price_long: Option<Price>,
+    pub price_short: Option<Price>,
+    pub min_quantity: Usd,
+    pub max_quantity: Usd,
+    pub tx_fee_rate: Option<TxFeeRate>,
+    pub funding_rate: Option<FundingRate>,
```

Comment:
Can we rename this to `daily_funding_rate` as per https://github.com/itchysats/itchysats/pull/1559?